README.md

```diff
@@ -1,12 +1,12 @@
 # Convox Praxis
 
-A framework for modern application infrastructure.
+Praxis is a framework for modern application infrastructure. It formalizes a set of portable infrastructure primitives that your application can use anywhere without changes, from your laptop to cloud infrastructure.
```

Comment:
Feels like `infrastructure` appears a few too many times. Maybe remove the last instances? 

"..., from your laptop to cloud."